.github/workflows/build_osgeo4w.sh

```diff
@@ -25,27 +25,28 @@ fi
 
 test -d "$1" && cd "$1"
 
-osgeo4w_path=/c/OSGeo4W64
+osgeo4w_path=/c/OSGeo4W
 arch=x86_64-w64-mingw32
 src=$(pwd)
 
 # compile
 
-export PATH=/mingw64/bin:/c/OSGeo4W64/bin:/usr/bin
-export PROJ_LIB=$osgeo4w_path/share/proj
+export PATH=/mingw64/bin:/c/OSGeo4W/bin:/usr/bin
 
```

Comment:
```suggestion
export C_INCLUDE_PATH=".:$osgeo4w_path/include:$src/dist.$arch/include"

```
OK, after adding -v to the gcc call, it became clear that the problem is that include paths are not converted to Windows style (like usually in MSYS). If include dirs are provided through an environment variable, headers are found (at least on my local MSYS)...

Thus the suggestion above...

Probably due to:
https://www.msys2.org/docs/filesystem-paths/